src/unit_tests/analysisd/test_decoder_syscollector.c

```diff
@@ -1617,25 +3227,18 @@ void test_syscollector_dbsync_network_iface_valid_msg_inserted(void **state)
 
     assert_int_not_equal(ret, 0);
 }
-void test_syscollector_dbsync_network_protocol_valid_msg_inserted(void **state)
+
+static const char *osinfo_empty_query = "agent 001 osinfo set NULL|NULL|NULL|NULL|NULL|NULL|"
```

Comment:
Coding style: we might move all the global declarations at the begging of the file to avoid the definition of variables in the middle of the tests.

Reply:
In this approach, try to follow the general guideline of declaring variables before their use, which facilitates a better understanding of the code. However, from now on it can be subject to debate.